Resources/Prototypes/Catalog/Fills/Crates/service.yml

```
@@ -423,4 +426,16 @@
         amount: 2
       - id: BoxCandleSmall
-        amount: 2
\ No newline at end of file
+        amount: 2
+- type: entity
+  id: CrateHolySupplies
+  parent: CratePlastic
+  name: holy supplies crate
+  description: Contains a Bible and 2 Jugs of holy water for religious needs
+  components:
+    - type: StorageFill
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)